db/upgrade/rdb_modern/tables/007-create_nrt_investigation.sql

```diff
@@ -94,8 +94,6 @@ CREATE TABLE dbo.nrt_investigation (
     cd                                  varchar(50)                                     NULL,
     cd_desc_txt                         varchar(100)                                    NULL,
     prog_area_cd                        varchar(20)                                     NULL,
-    jurisdiction_code                   varchar(20)                                     NULL,
-    jurisdiction_code_desc_txt          varchar(255)                                    NULL,
```

Comment:
Why `jurisdiction_code_desc_txt` is deleted from nrt? Correspondent column is the event DTO is `jurisdictionNm` and it's not deleted from the payload.

Reply:
jurisdiction_code_desc_txt is not used.

select count(*) from nrt_investigation;--13709
select count(*) from nrt_investigation where jurisdiction_code_desc_txt is null; ----13709

and jurisdiction_cd is the replacement for jurisdiction_code.

select sum (case when jurisdiction_cd = jurisdiction_code then 1 else 0 end) as 'j code match' from nrt_investigation ;